build.gradle.kts

```diff
@@ -170,6 +171,64 @@ subprojects {
     }
 }
 
+val byProjectNamePackageForVersionVariable: Map<String, String> = mapOf(
+    "model-server" to "org.modelix.model.server",
+    "modelql-core" to "org.modelix.modelql.core",
+)
```

Comment:
Can't we localize the logic more in the (two) modules that currently need it? You could define a utility task to generate such a version file and the two modules then call that task, passing in their required package name.

Reply:
I currently do not know how to do this and will need to take some time to read up on Gradle.

Reply:
Done